ckanext/justicehub_theme/templates/package/snippets/resources.html

```diff
@@ -0,0 +1,36 @@
+{#
+Displays a sidebard module with navigation containing the provided resources.
+If no resources are provided then the module will not be displayed.
+
+pkg       - The package dict that owns the resources.
+active    - The id of the currently displayed resource.
+action    - The controller action to use (default: 'resource_read').
+
+Example:
+
+  {% snippet "package/snippets/resources.html", pkg=pkg, active=res.id %}
+
+#}
+{% set resources = pkg.resources or [] %}
+{% if resources %}
+  {% block resources %}
+    <section class="module module-narrow">
+      {% block resources_inner %}
+        {% block resources_title %}
+          <h2 class="module-heading"><i class="fa fa-files-o"></i> {{ _("Resources") }}</h2>
+        {% endblock %}
+        {% block resources_list %}
+          <ul class="list-unstyled nav nav-simple rsrc-list-side" style="height:100vh;">
```

Comment:
let's avoid inline css as much possible. and also not set these specific heights wherever possible.